apisix/core/log.lua

```diff
@@ -15,13 +15,13 @@
 -- limitations under the License.
 --
 
-local ngx = ngx
-local ngx_log  = ngx.log
-local require  = require
-local select = select
+local ngx          = ngx
+local ngx_log      = ngx.log
+local require      = require
+local select       = select
```

Comment:
Please avoid changing the irrelevant code

Reply:
OK